PixelHunter1995/Coord.cs

```diff
@@ -13,6 +13,23 @@ public Coord(float x, float y)
             Y = y;
         }
 
+        public override bool Equals(object other)
+        {
+            Coord otherCoord = other as Coord;
+
+            if (otherCoord == null)
+            {
+                return false;
+            }
+
+            return this.X == otherCoord.X && this.Y == otherCoord.Y;
```

Comment:
If I don't remember incorrectly, this kind of equality check (exact equals) always tends to fail for floats (due to floating-point errors). Might suffice for our purposes (since they originate from the parser), but if coords starts being results from computations this is likely to break.

Reply:
Good point, will fix!